### PR TITLE
TSDEV-80 Whitelabel mobile browser intercept screens

### PR DESCRIPTION
**Pull request by thomchop**

### CODE REVIEW

**Comment by erikgui** [comment]
I updated the CSS styling and tested on my iPhone 6 with the following results:
landscape
portrait

The reason I need to use `height: 90vh` is because of the following issue with mobile Safari and possibly other mobile browsers:
https://nicolas-hoizey.com/2015/02/viewport-height-is-taller-than-the-visible-part-of-the-document-in-some-mobile-browsers.html

File: model/config.go [hunk 1]
```diff
@@ -307,6 +307,7 @@ type NativeAppSettings struct {
 	AppDownloadLink        *string
 	AndroidAppDownloadLink *string
 	IosAppDownloadLink     *string
+	IosAppUrlScheme        *string
```
Comment (csduarte) [inline model/config.go]: Doesn't android also have a Url Scheme?

File: webapp/components/get_android_app/get_android_app.jsx [hunk 2]
```diff
@@ -4,71 +4,33 @@
...
-                        <span className='get-android-app__app-creator'>
-                            <FormattedMessage
-                                id='get_app.mattermostInc'
-                                defaultMessage='uChat, Inc'
-                            />
-                        </span>
-                    </div>
-                </div>
+                <img
+                    src='https://s3.amazonaws.com/uber-test/uchat/uchat_color.png'
```
Comment (csduarte) [inline webapp/components/get_android_app/get_android_app.jsx]: is the best place for this image?

Comment (csduarte) [inline webapp/components/get_android_app/get_android_app.jsx]: i'd prefer something along the lines of http://[uchat url]/q/uchat_color.png

we would just need to upload the image to our stash of static files

File: webapp/components/get_ios_app/get_ios_app.jsx [hunk 3]
```diff
@@ -4,64 +4,35 @@
...
-                    className='get-ios-app__app-store-link'
-                    href={global.window.mm_config.IosAppDownloadLink}
-                    rel='noopener noreferrer'
-                >
-                    <img src={AppStoreButton}/>
-                </a>
                 <img
-                    className='get-app__screenshot'
-                    src={IPhone6Mockup}
+                    src='https://s3.amazonaws.com/uber-test/uchat/uchat_color.png'
```
Comment (csduarte) [inline webapp/components/get_ios_app/get_ios_app.jsx]: location change needed as well

**Comment by erikgui** [comment]
I am not sure if there's something I need to do to mark the changes requested as completed, but this PR is ready for merge.

**Comment by csduarte** [comment]
great work @thomchop @erikgui

**Pull request merged**